entrypoint.sh

```diff
@@ -21,48 +21,43 @@ fi
 # Run black with reviewdog
 black_exit_val="0"
 reviewdog_exit_val="0"
-if [[ "${INPUT_ANNOTATE,,}" = 'true' ]]; then
-  if [[ "${INPUT_REPORTER}" = 'github-pr-review' ]]; then
-    echo "[action-black] Checking python code with the black formatter and reviewdog..."
-    black_check_output=$(black --diff --quiet --check ${black_args[@]}) ||
-      black_exit_val="$?"
+if [[ "${INPUT_REPORTER}" = 'github-pr-review' ]]; then
+  echo "[action-black] Checking python code with the black formatter and reviewdog..."
+  black_check_output="$(black --diff --quiet --check ${black_args[*]})" ||
+    black_exit_val="$?"
 
-    # Intput black formatter output to reviewdog
-    echo "${black_check_output}" | reviewdog -f="diff" \
-      -f.diff.strip=0 \
-      -name="${INPUT_TOOL_NAME}-fix" \
-      -reporter="github-pr-review" \
-      -filter-mode="diff_context" \
-      -level="${INPUT_LEVEL}" \
-      -fail-on-error="${INPUT_FAIL_ON_ERROR,,}" \
-      ${INPUT_REVIEWDOG_FLAGS} || reviewdog_exit_val="$?"
-  else
+  # Intput black formatter output to reviewdog
+  echo "${black_check_output}" | reviewdog -f="diff" \
+    -f.diff.strip=0 \
+    -name="${INPUT_TOOL_NAME}-fix" \
+    -reporter="github-pr-review" \
+    -filter-mode="diff_context" \
+    -level="${INPUT_LEVEL}" \
+    -fail-on-error="${INPUT_FAIL_ON_ERROR,,}" \
+    ${INPUT_REVIEWDOG_FLAGS} || reviewdog_exit_val="$?"
+else
 
-    # Remove '-q' and '--quiet' form the black arguments
-    # NOTE: Having these flags in the action prevents the action from working.
-    black_args_tmp=()
-    for item in "${black_args[@]}"; do
-      if [[ "${item}" != "-q" && "${item}" != "--quiet" ]]; then
-        black_args_tmp+=("${item}") #Quotes when working with strings
-      fi
-    done
+  # Remove '-q' and '--quiet' form the black arguments
```

Comment:
Not directly related to this p-r, but I'm not sure it's worth doing.
Can we just add a document?
I don't think it's good to remove arguments silently without any log and we can simplify the script if we can remove this logic.

Reply:
Good point! Having the action do things silently might confuse people. I will add a comment to the documentation. Removeing these lines enables me to clean up the script a bit.

Reply:
fixed in 9eb9f9e